<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyPropertiesDlg</class>
 <widget class="QDialog" name="CopyPropertiesDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="CopyPropertiesUi" name="propertiesWidget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>182</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>DiskMaster options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="OptionWidget" name="DeviceOptionsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Copy options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="fullCopyButton">
              <property name="text">
               <string>Full copy</string>
              </property>
              <property name="icon">
               <iconset resource="diskmasterui.qrc">
                <normaloff>:/DiskMasterUI/icons/hard_drive_clone.png</normaloff>:/DiskMasterUI/icons/hard_drive_clone.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="partitionCopyButton">
              <property name="text">
               <string>Partition copy</string>
              </property>
              <property name="icon">
               <iconset resource="diskmasterui.qrc">
                <normaloff>:/DiskMasterUI/icons/Partition.png</normaloff>:/DiskMasterUI/icons/Partition.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="userCopyButton">
              <property name="text">
               <string>User copy</string>
              </property>
              <property name="icon">
               <iconset resource="diskmasterui.qrc">
                <normaloff>:/DiskMasterUI/icons/user_copy.png</normaloff>:/DiskMasterUI/icons/user_copy.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pagefullCopy">
            <layout class="QVBoxLayout" name="verticalLayout_2"/>
           </widget>
           <widget class="QWidget" name="pagePartitionCopy">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="PartitionTreeWidget" name="partitionWidget" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageUserCopy">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="CopyRangeWidget" name="rangeWidget" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopyPropertiesUi</class>
   <extends>QWidget</extends>
   <header>copypropertiesui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CopyRangeWidget</class>
   <extends>QWidget</extends>
   <header>copyrangewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PartitionTreeWidget</class>
   <extends>QWidget</extends>
   <header>partitiontreewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OptionWidget</class>
   <extends>QWidget</extends>
   <header>optionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="diskmasterui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CopyPropertiesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
